examples/chat/README.md

```diff
@@ -5,13 +5,19 @@ This directory holds a very simple chat application.
 All client participants connect to the same Oak server application (one client
 creates the application, the other clients connect to the created address:port).
 
-A client can then create a **room**; which returns a pair of random IDs that act
-as bearer tokens:
-
-- The **room ID** is needed to join that room and participate in chats there;
-  this is printed (as a base-64 string) to allow sharing.
-- The **admin ID** allows destruction of the room instance at the server; the
-  client keeps hold of this ID internally.
+A client can then create a **room**, which generates and returns a pair of
+random bearer tokens:
+
+- the **room access token** is needed to join that room and participate in chats
```

Comment:
Just checking: are this and "room admin token" in addition to the authentication via challenge-response mechanism?